src/inmanta/docstringParser.py

```diff
@@ -0,0 +1,63 @@
+"""
+    Copyright 2020 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from typing import Dict, Optional
+
+import docstring_parser
+
+
+class DocString:
+    def __init__(self, doc_string: docstring_parser.Docstring):
+        for attr in doc_string.meta:
+            # attr.args contains the part of an attribute definition in the docstring between the
+            # colons splitted on white spaces. This check enforces the format :attr <attribute-name>:
+            if len(attr.args) != 2 or attr.args[0] != "attr":
+                raise Exception(f"Failed to parse attribute {' '.join(attr.args)}")
```

Comment:
hard failure is perhaps not the best option here, could we log a warning and continue, so we only lose the bad one? 

(or make hard failure optional)